zingolib/src/wallet/notes.rs

```diff
@@ -79,6 +80,11 @@ impl AnyPoolOutput {
             )
             .collect()
     }
+    /// Uses a query to select all notes with specific properties and returns
+    /// a vector packing them in the AnyPoolOutput
+    pub fn get_all_outputs(query: &TransactionRecord) -> Vec<Self> {
+        Self::get_outputs_by_status(query, OutputSpendStatusQuery::any())
+    }
```

Comment:
this function is confused. its parameter is a TransactionRecord named query and though the comment describes a filter, there is none

Reply:
Addressing...